src/engine/controls/cuecontrol.cpp

```diff
@@ -1889,6 +1888,7 @@ void HotcueControl::slotHotcueGotoAndStop(double v) {
 }
 
 void HotcueControl::slotHotcueActivate(double v) {
+    qCDebug(QLoggingCategory("macros")) << "hotcue" << m_iHotcueNumber + 1 << "activated:" << v;
```

Comment:
You probably committed this by accident, please remove.

Reply:
Nope, that was intentional, it can ease debugging a lot. I use the logging category so it can easily be disabled.

Reply:
I think sooner or later you accidentally add it when doing a commit, and then it ends up in a totally unrelated commit. Yes, I also selectively commit all the time, but I really hate permanent uncommitted changes in my tree. Also, once you mark this file viewed, it won't be shown again unless it is changed ;)